gdc_client/download/client.py

```diff
@@ -276,27 +275,14 @@ def download_small_groups(self, smalls):
         errors = []
         groupings_len = len(smalls)
 
-        for i, s in enumerate(smalls):
+        for i, s in tqdm(iterable=enumerate(smalls), total=len(smalls),
```

Comment:
The progress bar above looks like it will still go from zero to complete for the small groups, since all the download event happen in this function. It might make sense to target the progress bar in the actual download function (self._download_tarfile(s)) instead so it captures the actual progress of the download